mu/app.py

```diff
@@ -42,12 +42,21 @@ def setup_logging():
     print('Logging to {}'.format(LOG_FILE))
 
 
+def excepthook(*exc_args):
+    """
+    Log exception and exit cleanly.
+    """
+    logging.error('Unrecoverable error', exc_info=(exc_args))
```

Comment:
Might be useful to also propagate the exception through the normal route, that way we can still see the trace from running from the python source or executable command line:
```
sys.__excepthook__(exc_args)
```